clients/drcachesim/tests/invariant_checker_test.cpp

```diff
@@ -176,7 +176,7 @@ bool
 check_branch_target_after_branch()
```

Comment:
x86-32 failed in what looks like something related to this PR:
```
356:   *** postcmd2 failed (Subprocess aborted): Trace invariant failure in T23655
356:   at ref # 438464 (1 instrs since timestamp 2057181375): Signal handler
356:   return point incorrect
356: 
356:   ***
355/415 Test #356: code_api|tool.histogram.offline ..................................***Failed    4.96 sec
```